compiler/quilt/data.py

```diff
@@ -131,17 +131,16 @@ def find_module(fullname, path=None):
 
                 # Try A Team/Other-Registry Path
                 team, user = parts
-                file_path = store.user_path(parts[0], parts[1])
-                if os.path.isdir(file_path):
-                    return FakeLoader(file_path)
+                user_path = store.user_path(team, user)
+                if os.path.isdir(user_path):
+                    return FakeLoader(user_path)
 
             raise ImportError('Could not find package by user {user!r} named {package!r}.\n  '
```

Comment:
Can you check that the error message looks right if the package doesn't exist? I think it's currently broken.